test/SampleTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Eris;
+
+class SampleTest extends \PHPUnit_Framework_TestCase
```

Comment:
probably you have seen https://github.com/giorgiosironi/eris/blob/master/examples/GeneratorSamplesTest.php which is an end2end test for this method instead

Reply:
I must admit that I didn't pay too much attention to the tests in the examples folder, because many of them are designed to fail. I found them very useful while learning about Eris, but since then I haven't really looked back.